tests/test_database_matrix.py

```diff
@@ -403,7 +402,7 @@ async def test_default_update_same_key(patched_send, opsdroid_matrix):
                 "/_matrix/client/r0/rooms/%21notaroomid/state/dev.opsdroid.database/twim",
                 {"twim": {"hello": "bob"}},
             ),
-        ],
+        ]
```

Comment:
I assume this and others are differerences in black config or version - broader issue but is it worth having some default config in the repo for consistency here? (if we don't already.)

Reply:
:shrug: I assume black did this, so I have no idea.